routers/repo/projects.go

```diff
@@ -416,21 +410,19 @@ func AddBoardToProjectPost(ctx *context.Context, form auth.EditProjectBoardTitle
 	})
 }
 
-// EditProjectBoardTitle allows a project board's title to be updated
-func EditProjectBoardTitle(ctx *context.Context, form auth.EditProjectBoardTitleForm) {
-
+func checkProjectBoardChangePermissions(ctx *context.Context) (*models.Project, *models.ProjectBoard) {
```

Comment:
do we have any tests for this function (or the APIs in general)? 

It looks like this is enforcing permission checks - it could be helpful to have these enforced at testing.